ci/jenkins/test.sh

```diff
@@ -545,7 +551,7 @@ function deliver_antrea_windows_containerd {
     echo "===== Deliver Antrea Windows to Windows worker nodes and pull necessary images on Windows worker nodes ====="
     sed -i 's/if (!(Test-Path $AntreaAgentConfigPath))/if ($true)/' hack/windows/Helper.psm1
     kubectl get nodes -o wide --no-headers=true | awk -v role="$CONTROL_PLANE_NODE_ROLE" '$3 !~ role && $1 ~ /win/ {print $1}' | while read WORKER_NAME; do
-        revert_snapshot_windows ${WORKER_NAME}
+        revert_snapshot_windows ${WORKER_NAME} win-initial-userspaceovs
```

Comment:
Do we plan to run all Windows tests with OVS userspace running inside containers?

Reply:
This change is temporary to run windows CI tests. @XinShuYang can confirm if we want to keep this change or remove after review and before merge.

Reply:
@Atish-iaf I think what Wenying means is that changing the Windows snapshot close to the 1.13 release date may introduce risks. You can use the win-initial-userspaceovs snapshot for CI verification, but restore the original snapshot name before merging the PR. We can create a new PR to update the Windows snapshot after 1.13 release.